package.json

```diff
@@ -18,7 +18,13 @@
   "license": "ISC",
   "dependencies": {
     "body-parser": "^1.19.0",
+    "connect-ensure-login": "^0.1.1",
```

Comment:
Lets remove this line and then this looks good to me

Reply:
As mentioned above this is module is currently required as written.

Reply:
Okay I was able to remove this dependency in 917eb19 do you mind reviewing these changes locally and merging?

Reply:
How are you running this btw?  Does running `npm run start:dev` work on your machine?